stock_move_free_reservation_reassign/models/stock_quant.py

```diff
@@ -25,5 +25,7 @@ def _apply_inventory(self):
         ]._get_move_free_reservation_ids() as move_to_reassign_ids:
             res = super()._apply_inventory()
         if move_to_reassign_ids:
-            self.env["stock.move"].browse(move_to_reassign_ids)._action_assign()
+            moves = self.env["stock.move"].browse(move_to_reassign_ids)
+            if not self.env.context.get("assign_in_inventory_mode"):
```

Comment:
@nguyenminhchien I don't get why you need a second context parameter.

The ir.config_parameter is sufficent.

Reply:
it's painful to add the context "assign_in_inventory_mode". Remove inventory_mode (`inventory_mode=False`) is a must to keep this module works, making the current tests green are enough to explain the change.
So, i will revert this change, no need more unittest, what do you think?